tendrl/notifier/manager/__init__.py

```diff
@@ -25,7 +25,7 @@ def __init__(self):
             Event(
                 ExceptionMessage(
                     priority="debug",
-                    publisher="alerting",
+                    publisher="notifier",
```

Comment:
i forgot to change publisher from alerting to notifier, so @TimothyAsir changed in all places